--- lib/admin.defines.php	2009-11-02 09:07:56.000000000 +0100
+++ lib/admin.defines__vt.php	2009-11-02 18:21:48.000000000 +0100
@@ -53,7 +53,7 @@
 include_once (FSROOT."lib/mail/class.phpmailer.php");
 
 // INCLUDE MISC
-include (FSROOT."lib/Misc.php");
+include (FSROOT."lib/Misc__vt.php");
 include (dirname(__FILE__)."/Class.NotificationsDAO.php");
 include (dirname(__FILE__)."/Class.Notification.php");
 include (dirname(__FILE__)."/Class.Mail.php");
@@ -88,8 +88,11 @@
 // Define a demo mode
 define("DEMO_MODE", false);
 
+// Define VT mode
+define("VT_MODE", true);
+
 // Parameter to show link to Asterisk GUI
-define("ASTERISK_GUI_LINK", false);
+define("ASTERISK_GUI_LINK", true);
 
 
 define ("LEN_ALIASNUMBER", isset($A2B->config['global']['len_aliasnumber'])?$A2B->config['global']['len_aliasnumber']:null);
@@ -205,7 +208,7 @@
 // INCLUDE HELP
 include (LIBDIR."admin.help.php");
 
-include (LIBDIR."common.defines.php");
+include (LIBDIR."common.defines__vt.php");
 
 
 define ("RELOAD_ASTERISK_IF_SIPIAX_CREATED", isset($A2B->config["signup"]['reload_asterisk_if_sipiax_created'])?$A2B->config["signup"]['reload_asterisk_if_sipiax_created']:0);
