--- lib/common.defines.php	2009-10-23 12:43:07.000000000 +0200
+++ lib/common.defines__vt.php	2009-11-02 11:16:23.000000000 +0100
@@ -114,14 +114,14 @@
 
 // A2BILLING INFO
 define ("WEBUI_DATE", 'Released : 02 October 2009');
-define ("WEBUI_VERSION", 'A2Billing - Version 1.4.2.1 (Radix)');
+define ("WEBUI_VERSION", 'A2Billing - Version 1.4.2.1 (Radix - Village Telco Edition)');
 
 // A2BILLING COPYRIGHT & CONTACT
 define ("TEXTCONTACT", gettext("This software has been created by Areski under AGPL licence. For futher information, feel free to contact me:"));
 define ("EMAILCONTACT", "areski@gmail.com");
 
   
-define ("COPYRIGHT", "A2Billing - Version 1.4.2.1 (Radix) - Released : 02 October 2009, A2Billing is software licensed under the ".'<a href="http://www.fsf.org/licensing/licenses/agpl-3.0.html" target="_blank">AGPL 3</a>'.". <br/>" .
+define ("COPYRIGHT", "A2Billing - Version 1.4.2.1 (Radix -- Village Telco Edition) - Released : 02 October 2009, A2Billing is software licensed under the ".'<a href="http://www.fsf.org/licensing/licenses/agpl-3.0.html" target="_blank">AGPL 3</a>'.". <br/>" .
 		"Copyright (C) 2004-2009 - Star2billing S.L. <a href=\"http://www.star2billing.com\" target=\"_blank\">http://www.star2billing.com/</a>");
 
 define ("CCMAINTITLE", gettext("A2Billing Portal"));
