--- lib/Misc.php	2009-11-03 17:36:55.000000000 +0100
+++ lib/Misc__vt.php	2009-11-02 18:22:46.000000000 +0100
@@ -487,7 +487,7 @@
 	$list_customer = $inst_table->Get_list($handle, $FG_TABLE_CLAUSE, "", "", "", "", "", "", "", 10);
 	$id = $list_customer[0][0];
 	if ($id > 0) {
-		echo "<a href=\"A2B_entity_card.php?form_action=ask-edit&id=$id\">$value</a>";
+		echo "<a href=\"A2B_entity_card__vt.php?form_action=ask-edit&id=$id\">$value</a>";
 	} else {
 		echo $value;
 	}
@@ -1338,31 +1338,3 @@
     return round($time_end - $mt_time, $len);
 }
 
-
-
-/*
-   * @return string
-   * @param string $url
-   * @desc Return string content from a remote file
-*/
-
-function open_url($url)
-{
-    $ch = curl_init();
-
-    curl_setopt ($ch, CURLOPT_URL, $url);
-    curl_setopt ($ch, CURLOPT_HEADER, 0);
-
-    ob_start();
-
-    curl_exec ($ch);
-    curl_close ($ch);
-    $string = ob_get_contents();
-
-    ob_end_clean();
-   
-    return $string;    
-}
-
-
-
