diff -urN ../A2Billing_UI/lib/didx.php A2Billing_UI/lib/didx.php
--- ../A2Billing_UI/lib/didx.php	1969-12-31 19:00:00.000000000 -0500
+++ A2Billing_UI/lib/didx.php	2008-02-28 21:09:49.000000000 -0500
@@ -0,0 +1,231 @@
+<?php
+define(DIDX_ID, "700XXX");
+define(DIDX_PASS, "password");
+define(DIDX_SITE, "didx.net");
+define(RING_TO, "your.domain.com");
+define(MIN_RATING, 1);
+
+class didx {
+	function setupcost($cost) {
+		if($cost !== "N/A")
+			return $cost/*+ add your profit here*/;
+		else
+			return $cost;
+	}
+	function monthly($cost) {
+		if($cost !== "N/A")
+			return $cost/*+ add your profit here*/;
+		else
+			return $cost;
+	}
+	function minutecharge($cost) {
+		if($cost !== "N/A")
+			return $cost/*+ add your profit here*/;
+		else
+			return $cost;
+	}
+	function freeminutes($mins) {
+		return $mins/*+ add your profit here*/;
+	}
+
+	function getAvailableDIDS($country, $area, $vendor=-1, $VRatingFrom=MIN_RATING, $VRatingTo=10, $CountryID=-1) {
+		$ret = true;
+		$client = new SoapClient(null, array('location' => "https://".DIDX_SITE."/cgi-bin/WebGetListServer.cgi",
+						     'connection_timeout' => 5,
+		                                     'uri'      => "http://".DIDX_SITE."/GetList"));
+		try {		     
+			$ret = $client->getAvailableDIDS(DIDX_ID, DIDX_PASS, $country, $area, $vendor, $VRatingFrom, $VRatingTo, $CountryID);
+			$ret = $ret['Array'];
+			@sort($ret);
+			for($i=1; $i<count($ret); $i++) {
+				$ret[$i][1]=$this->setupcost($ret[$i][1]);
+				$ret[$i][2]=$this->monthly($ret[$i][2]);
+				$ret[$i][3]=$this->minutecharge($ret[$i][3]);
+			}
+		} catch (SoapFault $exception) {
+			$ret = false;
+		} catch (Exception $e) {
+			$ret = false;
+		}
+		return $ret;
+	}
+
+	function getAvailableRatedDIDSbyCountryCode ($country, $VRatingFrom=MIN_RATING,$VRatingTo=10, $limit=-1, $vendors=-1, $ratefrom=-1, $rateto=-1, $CountryID=-1) {
+		$ret = true;
+		$client = new SoapClient(null, array('location' => "https://".DIDX_SITE."/cgi-bin/WebGetAllRatedDIDS.cgi",
+						'trace'=>1,
+						     'connection_timeout' => 5,
+		                                     'uri'      => "http://".DIDX_SITE."/GetList"));
+		try {		     
+			$ret = $client->getAvailableRatedDIDSbyCountryCode(DIDX_ID, DIDX_PASS, $country, $VRatingFrom, $VRatingTo, $limit, $vendors, $ratefrom, $rateto, $CountryID);
+			$ret = $ret['Array'];
+			sort($ret);
+			for($i=1; $i<count($ret); $i++) {
+				$ret[$i][1]=$this->setupcost($ret[$i][1]);
+				$ret[$i][2]=$this->monthly($ret[$i][2]);
+				$ret[$i][3]=$this->minutecharge($ret[$i][3]);
+			}
+		} catch (SoapFault $exception) {
+			$ret = false;
+		} catch (Exception $e) {
+			$ret = false;
+		}
+		return $ret;
+	}
+
+	function getDIDCountry($VRatingFrom=MIN_RATING,$VRatingTo=10) {
+		$ret = true;
+		if(isset($_SESSION["country-$VRatingFrom"]))
+			return $_SESSION["country-$VRatingFrom"];
+		$client = new SoapClient(null, array('location' => "https://".DIDX_SITE."/cgi-bin/WebGetDIDCountriesServer.cgi",
+						     'connection_timeout' => 5,
+		                                     'uri'      => "http://".DIDX_SITE."/GetList"));
+		try {		     
+			$ret = $client->getDIDCountry(DIDX_ID, DIDX_PASS, $VRatingFrom, $VRatingTo);
+			$ret = $ret['Array'];
+			$_SESSION["country-$VRatingFrom"] = $ret;
+		} catch (SoapFault $exception) {
+			$ret = false;
+		} catch (Exception $e) {
+			$ret = false;
+		}
+		return $ret;
+	}
+	
+	function getDIDArea($CountryCode, $VRatingFrom=MIN_RATING,$VRatingTo=10, $vendors="", $ratefrom=-1, $rateto=-1, $CountryID=-1) {
+		$ret = true;
+		if(isset($_SESSION["country-$CountryCode-$CountryID-$VRatingFrom"]))
+			return $_SESSION["country-$CountryCode-$CountryID-$VRatingFrom"];
+		$client = new SoapClient(null, array('location' => "https://".DIDX_SITE."/cgi-bin/WebGetDIDAreasServer.cgi",
+						     'connection_timeout' => 5,
+						'trace'=>1,
+		                                     'uri'      => "http://".DIDX_SITE."/GetList"));
+		try {		     
+			$ret = $client->getDIDArea(DIDX_ID, DIDX_PASS, $CountryCode, $VRatingFrom, $VRatingTo, $vendors, $ratefrom, $rateto, $CountryID);
+			$ret = $ret['Array'];
+			$_SESSION["country-$CountryCode-$CountryID-$VRatingFrom"] = $ret;
+		} catch (SoapFault $exception) {
+			$ret = false;
+		} catch (Exception $e) {
+			$ret = false;
+		}
+		return $ret;
+	}
+	
+	function GetCostOfDIDByNumber($DIDNumber) {
+		$client = new SoapClient(null, array('location' => "https://".DIDX_SITE."/cgi-bin/WebGetCostOfDIDServer.cgi",
+		                                     'uri'      => "http://".DIDX_SITE."/GetCost"));
+		try {		     
+			$ret = $client->GetCostOfDIDByNumber(DIDX_ID, DIDX_PASS, $DIDNumber);
+			$ret = $ret['Array'][1];
+		} catch (SoapFault $exception) {
+			$ret = false;
+		} catch (Exception $e) {
+			$ret = false;
+		}
+		return $ret;
+	}
+
+	function BuyDIDByNumber($DIDNumber,$SIPorIAX, $Flag=1,$VendorID=-1) {
+		
+		$ret = true;
+		$client = new SoapClient(null, array('location' => "https://".DIDX_SITE."/cgi-bin/WebBuyDIDServer.cgi",
+						'trace'=>1,
+		                                     'uri'      => "http://".DIDX_SITE."/BuyDID"));
+		try {		     
+			$ret = $client->BuyDIDByNumber(DIDX_ID, DIDX_PASS, $DIDNumber, $SIPorIAX, $Flag,$VendorID);
+		} catch (SoapFault $exception) {
+			$ret = -99;
+		} catch (Exception $e) {
+			$ret = -99;
+		}
+		return $ret;
+	}
+
+	function  ReleaseDID($DIDNumber) {
+		
+		$ret = true;
+		$client = new SoapClient(null, array('location' => "https://".DIDX_SITE."/cgi-bin/WebReleaseDIDServer.cgi",
+						'trace'=>1,
+		                                     'uri'      => "http://".DIDX_SITE."/Release"));
+		try {		     
+			$ret = $client-> ReleaseDID(DIDX_ID, DIDX_PASS, $DIDNumber);
+		} catch (SoapFault $exception) {
+			$ret = -99;
+		} catch (Exception $e) {
+			$ret = -99;
+		}
+		return $ret;
+	}
+
+	function getDIDMinutesInfo($DIDNumber) {
+		$client = new SoapClient(null, array('location' => "http://".DIDX_SITE."/cgi-bin/WebGetDIDSMinutes.cgi",
+						'trace'=>1,
+		                                     'uri'      => "http://".DIDX_SITE."/GetMinutes"));
+		try {		     
+			$ret = $client->getDIDMinutesInfo(DIDX_ID, DIDX_PASS, $DIDNumber);
+			$ret = $ret['Array'][1];
+		} catch (SoapFault $exception) {
+			$ret = false;
+		} catch (Exception $e) {
+			$ret = false;
+		}
+		$cost = $this->GetCostOfDIDByNumber($DIDNumber);
+		$ret = array("Setup Price"=> $this->setupcost($cost[0]),
+		        "Monthly Price" => $this->monthly($cost[1]),
+		        "Minutes Included" => $this->freeminutes($ret[1]),
+		        "After free Minute Charges" => $this->minutecharge($ret[2]),
+		        "Channels" => $ret[3]
+		        );
+		return $ret;
+	}
+
+	function getAvailableRatedNXX($CountryCode, $NPA, $VRatingFrom=MIN_RATING,$VRatingTo=10, $Vendor=-1) {
+		if(isset($_SESSION["country-$CountryCode-$NPA-$VRatingFrom"]))
+			return $_SESSION["country-$CountryCode-$NPA-$VRatingFrom"];
+		$client = new SoapClient(null, array('location' => "https://".DIDX_SITE."/cgi-bin/WebGetListServer.cgi",
+						'trace'=>1,
+		                                     'uri'      => "http://".DIDX_SITE."/GetList"));
+		try {		     
+			$ret = $client->getAvailableRatedNXX(DIDX_ID,DIDX_PASS,$CountryCode,$NPA,$VRatingFrom,$VRatingTo, $Vendor);
+			$ret = $ret['Array'];
+			$_SESSION["country-$CountryCode-$NPA-$VRatingFrom"] = $ret;
+		} catch (SoapFault $exception) {
+			$ret = false;
+		} catch (Exception $e) {
+			$ret = false;
+		}
+		return $ret;
+	}
+	
+	function getDIDCallLog($DIDNumber, $page=0) {
+		$client = new SoapClient(null, array('location' => "http://".DIDX_SITE."/cgi-bin/WebGetDIDUsage.cgi",
+						'trace'=>1,
+		                                     'uri'      => "http://".DIDX_SITE."/GetList"));
+		try {		     
+			$ret = $client->getDIDCallLog(DIDX_ID, DIDX_PASS, $DIDNumber, $page);
+			$ret = $ret['Array'];
+		} catch (SoapFault $exception) {
+			$ret = false;
+		} catch (Exception $e) {
+			$ret = false;
+		}
+		return $ret;
+	}
+
+	function DoTestMyPurchasedDID($DIDNumber, $sMail=2 ) {
+		$client = new SoapClient(null, array('location' => "https://".DIDX_SITE."/cgi-bin/WebDIDTester.cgi",
+						'trace'=>1,
+		                                     'uri'      => "http://".DIDX_SITE."/DIDTest"));
+		try {		     
+			$ret = $client->DoTestMyPurchasedDID(DIDX_ID, DIDX_PASS, $DIDNumber, $sMail);
+		} catch (SoapFault $exception) {
+			$ret = $exception;
+		} catch (Exception $e) {
+			$ret = $e;
+		}
+		return $ret;
+	}
+}
+
+?>
diff -urN ../A2Billing_UI/Public/A2B_entity_didx.php A2Billing_UI/Public/A2B_entity_didx.php
--- ../A2Billing_UI/Public/A2B_entity_didx.php	1969-12-31 19:00:00.000000000 -0500
+++ A2Billing_UI/Public/A2B_entity_didx.php	2008-02-28 20:57:03.000000000 -0500
@@ -0,0 +1,231 @@
+<?php
+include ("../lib/defines.php");
+include ("../lib/module.access.php");
+include ("../lib/Form/Class.FormHandler.inc.php");
+include ("./form_data/FG_var_did.inc");
+include ("../lib/smarty.php");
+require ("../lib/didx.php");
+
+if (! has_rights (ACX_DID)){
+	   Header ("HTTP/1.0 401 Unauthorized");
+	   Header ("Location: PP_error.php?c=accessdenied");
+	   die();
+
+}
+
+/***********************************************************************************/
+function unavailable() {
+	return "Sorry, the DID selection service is currently unavailable because of uplink error, please come back and try again later or contact us about available DIDs (include country and area codes to your request).";
+}
+
+function getcountry($arr, $selected=0, $id=0) {
+	if($arr === false)
+		return unavailable();
+	$res = "<select name=\"country\" onchange=\"this.form.elements['area'].value=0; this.form.elements['nxx'].value=0; this.form.elements['number'].value=0; this.form.submit();\">\n";
+	$res .= "<option value=\"0\">Select a Country</option>\n";
+	if($arr) {
+		$i = 0;
+		foreach($arr as $c) {
+			if($i++ == 0)
+				continue;
+			$res .= "<option value=\"$c[1]/$c[2]/$c[0]\"";
+			if($c[1] == $selected && $c[2] == $id)
+				$res .= " selected";
+			$res .= ">$c[0] ($c[1])</option>\n";
+		}
+	}
+	$res .= "</select>";
+	return $res;
+}
+
+function getarea($arr, $selected=0) {
+	if($arr === false)
+		return unavailable();
+	$res = "<select name=\"area\" onchange=\"this.form.elements['nxx'].value=0; this.form.elements['number'].value=0; this.form.submit();\">\n";
+	$res .= "<option value=\"0\">Select an area code</option>\n";
+	if($arr) {
+		$i = 0;
+		foreach($arr as $c) {
+			if($i++ == 0)
+				continue;
+			$c[1] = iconv("ISO-8859-1", "UTF-8", $c[1]);
+			$res .= "<option value=\"$c[0]\"";
+			if($c[0] == $selected)
+				$res .= " selected";
+			$res .= ">$c[0] - $c[1]</option>\n";
+		}
+	}
+	$res .= "</select>";
+	return $res;
+}
+
+function getnxx($arr, $selected=0) {
+	if($arr === false)
+		return unavailable();
+	$res = "<select name=\"nxx\" onchange=\"this.form.elements['number'].value=0; this.form.submit();\">\n";
+	$res .= "<option value=\"0\">Select rate center</option>\n";
+	if($arr) {
+		$i = 0;
+		foreach($arr as $c) {
+			if($i++ == 0)
+				continue;
+			$res .= "<option value=\"$c[4]\"";
+			if($c[4] == $selected)
+				$res .= " selected";
+			$res .= ">1$c[2]$c[4]</option>\n";
+		}
+	}
+	$res .= "</select>";
+	return $res;
+}
+
+function getnumber($arr, $country, $selected=0) {
+	if($arr === false)
+		return unavailable();
+	$res = "<select name=\"number\" onchange=\"this.form.submit();\">\n";
+	$res .= "<option value=\"0\">Select a number</option>\n";
+	if($arr) {
+		$i = 0;
+		foreach($arr as $c) {
+			if($i++ == 0)
+				continue;
+			$res .= "<option value=\"$c[0]\"";
+			if($c[0] == $selected)
+				$res .= " selected";
+			$res .= ">$c[0] ($c[1]/$c[2])</option>\n";
+		}
+	}
+	$res .= "</select>";
+	return $res;
+}
+
+$HD_Form -> setDBHandler (DbConnect());
+
+ini_set("precision", "16");
+$didx = new didx();
+
+if($form_action == "purchase" || $form_action == "add") {
+	if($form_action == "purchase")
+		$form_action = "ask-add";
+	$HD_Form -> init();
+	$list = $HD_Form -> perform_action($form_action);
+	$smarty->display('main.tpl');
+
+        $res = $didx->BuyDIDByNumber($did,"$did@".RING_TO);
+        if($res < 0)
+                echo "Error $res while setting up the DID $did.";
+	else {			
+		echo "The DID $did successfully purchased on DIDX";
+		// #### TOP SECTION PAGE
+		$HD_Form -> create_toppage ($form_action);
+
+		// #### CREATE FORM OR LIST
+		//$HD_Form -> CV_TOPVIEWER = "menu";
+		if (strlen($_GET["menu"])>0) $_SESSION["menu"] = $_GET["menu"];
+
+		$HD_Form -> create_form ($form_action, $list, $id=null) ;
+	}
+} else {
+
+	$smarty->display('main.tpl');
+
+	echo $CC_help_list_did;
+
+
+	$country = 0;
+	$area = 0;
+	$nxx = 0;
+	$number = 0;
+	$ID = 0;
+	$rating = MIN_RATING;
+
+	if($_GET['country'])
+		list($country, $ID, $countryname) = sscanf($_GET['country'], "%d/%d/%[^[]]");
+	if($_GET['area'])
+		$area = $_GET['area'];
+	if($_GET['area'])
+		$nxx = $_GET['nxx'];
+	if($_GET['number'])
+		$number = $_GET['number'];
+	if($_GET['ID'])
+		$ID = $_GET['ID'];
+	if($_GET['rating'])
+		$rating = $_GET['rating'];
+	$country_arr=$area_arr=$nxx_arr=$number_arr=0;
+
+	$country_arr = $didx->getDIDCountry($rating);
+	if($country) {
+		$area_arr = $didx->getDIDArea($country, $rating, 10, "", "", "", $ID);
+	}
+	if($country && $area) {
+		if($country == 1)
+			$nxx_arr = $didx->getAvailableRatedNXX($country, $area, $rating);
+		if($country != 1 || $nxx) {
+			if($nxx == 0)
+				$nxx = '';
+			if($area < 0)
+				$area = '';
+			$number_arr = $didx->getAvailableRatedDIDSbyCountryCode($country.$area.$nxx, $rating ,10,20,"","","",$ID);
+		}
+	}
+?>
+<table width=100%>
+<tr><td>
+<FORM action="<?php echo $_SERVER['PHP_SELF']; ?> " method="get">
+<?php
+$res = "<select name=\"rating\" onchange=\"this.form.submit();\">\n";
+for($i=1; $i<=9;$i++) {
+	$res .= "<option value=\"$i\"";
+	if($i == $rating)
+		$res .= " selected";
+	$res .= ">".$i."</option>\n";
+}
+$res .= "</select>\n";
+echo $res;
+?>
+Minimal <a href="http://www.didx.net/rating" target="new">DID vendor rating</a>
+<br />
+<?php echo getcountry($country_arr, $country, $ID);?>
+<br />
+<?php echo getarea($area_arr, $area);?>
+<br />
+<?php if($country==1) echo getnxx($nxx_arr, $nxx)."<br />\n"; else echo '<input type="hidden" name="nxx" value="0">'; ?>
+<?php echo getnumber($number_arr, $country, $number);?>
+<br />
+</form>
+</td></tr>
+</table>
+<?php
+
+if($number) {
+	$vr = $didx->GetCostOfDIDByNumber($number);
+	$number_info = $didx->getDIDMinutesInfo($number);
+	$number_info['Vendor rating'] = $vr[3];
+	echo "<h3>DID $number parameters:</h3><br />";
+	echo "<ul>";
+	foreach($number_info as $key => $value) {
+		echo "<li>$key:&nbsp;$value</li>";
+	}
+	echo "</ul>";
+	$instance_table = new Table("cc_country");
+	if($countryname == "USA")
+		$countryname = "United States";
+	$QUERY = "select id from cc_country where countryname like '".$countryname."%'";
+	$countryinfo = $instance_table -> SQLExec ($HD_Form -> DBHandle, $QUERY);
+?>
+<form method="post" action="<?php echo $PHP_SELF; ?>">
+<input type="hidden" name="form_action" value="purchase">
+<input type="hidden" name="did" value="<?php echo $number; ?>">
+<input type="hidden" name="fixrate" value="<?php echo $number_info["Monthly Price"]; ?>">
+<input type="hidden" name="id_cc_country" value="<?php echo $countryinfo[0][0]; ?>">
+<input type="submit" name="submit" value="Buy DID <?php echo $number; ?>" style="border: 2px outset rgb(204, 51, 0);">
+</form>
+
+<?php
+	}
+}
+
+// #### FOOTER SECTION
+$smarty->display('footer.tpl');
+
+?>
diff -urN ../A2Billing_UI/Public/form_data/FG_var_did.inc A2Billing_UI/Public/form_data/FG_var_did.inc
--- ../A2Billing_UI/Public/form_data/FG_var_did.inc	2007-04-27 17:02:47.000000000 -0400
+++ A2Billing_UI/Public/form_data/FG_var_did.inc	2008-02-28 19:15:10.000000000 -0500
@@ -58,7 +58,7 @@
 
 
 // TODO integrate in Framework
-if ($form_action=="ask-add"){
+if ($form_action=="ask-add" || $form_action=="purchase"){
 	$begin_date = date("Y");
 	$begin_date_plus = date("Y")+25;
 	$end_date = date("-m-d H:i:s");
@@ -66,6 +66,11 @@
 	$comp_date_plus = "value='".$begin_date_plus.$end_date."'";
 }
 
+if ($form_action=="purchase"){
+	$didval = "value='".$did."'";
+	$fixval = "value='".$fixrate."'";
+}
+
 $did_regexpress = '9';
 if ($form_action=="ask-edit" || $form_action=="edit"){
 	$did_regexpress = '13';
@@ -75,7 +80,7 @@
 	"did",
 	'$value',
 	"INPUT",
-	"size=30 maxlength=30",
+	"size=30 maxlength=30 $didval",
 	"$did_regexpress",
 	gettext("Insert the DID"),
 	"" , "", "", "", "" , "", "" ,"Insert DID");
@@ -141,7 +146,7 @@
 	"fixrate",
 	'$value',
 	"INPUT",
-	"size=20  maxlength=20",
+	"size=20  maxlength=20 $fixval",
 	"12",
 	gettext("Insert the monthly rate"),
 	"" , "", "", "", "", "", "", gettext("currency : ").strtoupper(BASE_CURRENCY) );
diff -urN ../A2Billing_UI/Public/templates/default/main.tpl A2Billing_UI/Public/templates/default/main.tpl
--- ../A2Billing_UI/Public/templates/default/main.tpl	2007-08-06 09:06:44.000000000 -0400
+++ A2Billing_UI/Public/templates/default/main.tpl	2008-02-28 17:50:06.000000000 -0500
@@ -230,6 +230,7 @@
 				<li><a href="A2B_entity_didgroup.php?stitle=DID+Group&form_action=ask-add&section=5">{php} echo gettext("Add DID Group");{/php}</a></li>
 				<li><a href="A2B_entity_did.php?stitle=DID&section=5">{php} echo gettext("List DID");{/php}</a></li>
 				<li><a href="A2B_entity_did.php?stitle=DID&form_action=ask-add&section=5">{php} echo gettext("Add DID");{/php}</a></li>
+				<li><a href="A2B_entity_didx.php?stitle=DID">{php} echo gettext("Add DID from DIDX");{/php}</a></li>
                 <li><a href="A2B_entity_did_import.php?stitle=DID&section=5">{php} echo gettext("Import DID");{/php}</a></li>
 				<li><a href="A2B_entity_did_destination.php?stitle=DID+Destination&section=5">{php} echo gettext("List Destination");{/php}</a></li>
 				<li><a href="A2B_entity_did_destination.php?stitle=DID+Destination&form_action=ask-add&section=5">{php} echo gettext("Add Destination");{/php}</a></li>
