--- admin/Public/A2B_entity_voucher.php	2010-04-07 14:32:20.000000000 +0400
+++ vt_simplified_gui_1.7.0/admin/Public/A2B_entity_voucher__vt.php	2010-04-08 20:04:29.000000000 +0400
@@ -32,15 +32,15 @@
 **/
 
 
-include ("../lib/admin.defines.php");
-include ("../lib/admin.module.access.php");
-include ("../lib/Form/Class.FormHandler.inc.php");
-include ("./form_data/FG_var_voucher.inc");
-include ("../lib/admin.smarty.php");
+include ("../lib/admin.defines__vt.php");
+include ("../lib/admin.module.access__vt.php");
+include ("../lib/Form/Class.FormHandler.inc__vt.php");
+include ("./form_data/FG_var_voucher__vt.inc");
+include ("../lib/admin.smarty__vt.php");
 
 if (!has_rights(ACX_BILLING)) {
 	Header("HTTP/1.0 401 Unauthorized");
-	Header("Location: PP_error.php?c=accessdenied");
+	Header("Location: PP_error__vt.php?c=accessdenied");
 	die();
 }
 
@@ -130,14 +130,15 @@
 $list = $HD_Form->perform_action($form_action);
 
 // #### HEADER SECTION
-$smarty->display('main.tpl');
+$smarty->display('main__vt.tpl');
 
 // #### HELP SECTION
+if (!VT_MODE) {
 if ($form_action == 'list')
 	echo $CC_help_list_voucher;
 else
 	echo $CC_help_create_voucher;
-	
+}	
 
 
 ?>
@@ -145,8 +146,10 @@
 
 
 <div class="toggle_hide2show">
+<?php if (!VT_MODE) { ?> 
 <center><a href="#" target="_self" class="toggle_menu"><img class="toggle_hide2show" src="<?php echo KICON_PATH; ?>/toggle_hide2show.png" onmouseover="this.style.cursor='hand';" HEIGHT="16"> <font class="fontstyle_002"><?php echo gettext("SEARCH VOUCHERS");?> </font></a></center>
-	<div class="tohide" style="display:none;">
+<?php } ?>
+    <div class="tohide" style="display:none;">
 
 <?php
 // #### CREATE SEARCH FORM
@@ -171,8 +174,10 @@
 ?>
 <!-- ** ** ** ** ** Part for the Update ** ** ** ** ** -->
 <div class="toggle_hide2show">
+<?php if (!VT_MODE) { ?>
 <center><a href="#" target="_self" class="toggle_menu"><img class="toggle_hide2show" src="<?php echo KICON_PATH; ?>/toggle_hide2show.png" onmouseover="this.style.cursor='hand';" HEIGHT="16"> <font class="fontstyle_002"><?php echo gettext("BATCH UPDATE");?> </font></a></center>
-	<div class="tohide" style="display:none;">
+<?php } ?>
+    <div class="tohide" style="display:none;">
 
 <center>
 <b>&nbsp;<?php echo $HD_Form -> FG_NB_RECORD ?> <?php echo gettext("vouchers selected!"); ?>&nbsp;<?php echo gettext("Use the options below to batch update the selected vouchers.");?></b>
