--- admin/Public/A2B_notification.php	2010-04-07 14:32:20.000000000 +0400
+++ vt_simplified_gui_1.7.0/admin/Public/A2B_notification__vt.php	2010-04-08 20:04:29.000000000 +0400
@@ -32,13 +32,13 @@
 **/
 
 
-include ("../lib/admin.defines.php");
-include ("../lib/admin.module.access.php");
-include ("../lib/admin.smarty.php");
+include ("../lib/admin.defines__vt.php");
+include ("../lib/admin.module.access__vt.php");
+include ("../lib/admin.smarty__vt.php");
 
 if (!$ACXACCESS) {
 	Header("HTTP/1.0 401 Unauthorized");
-	Header("Location: PP_error.php?c=accessdenied");
+	Header("Location: PP_error__vt.php?c=accessdenied");
 	die();
 }
 
@@ -115,9 +115,10 @@
 
 
 // #### HEADER SECTION
-$smarty->display('main.tpl');
-
+$smarty->display('main__vt.tpl');
+if (!VT_DEMO) {
 echo $CC_help_notifications;
+} 
 $nb_by_page = 15;
 $nb_total = NotificationsDAO::getNbNotifications();
 $nb_page = ceil($nb_total/$nb_by_page);
@@ -144,14 +145,14 @@
 	<tr>
 		<td colspan="3" align="left">
 		<?php if($page>1){ ?>
-			<a href="A2B_notification.php?page=<?php echo $page-1; ?>"> &lt; <?php echo gettext("Newer") ?> </a>
+			<a href="A2B_notification__vt.php?page=<?php echo $page-1; ?>"> &lt; <?php echo gettext("Newer") ?> </a>
 		<?php } ?>
 		&nbsp;
 		</td>
 		<td colspan="3" align="right">
 		&nbsp;
                 <?php if($page<$nb_page){ ?>
-		<a href="A2B_notification.php?page=<?php echo $page+1; ?>"><?php echo gettext("Older") ?> &gt;</a>
+		<a href="A2B_notification__vt.php?page=<?php echo $page+1; ?>"><?php echo gettext("Older") ?> &gt;</a>
                 <?php } ?>
 		</td>
 	</tr>
@@ -228,14 +229,14 @@
 		<tr>
 			<td colspan="3" align="left">
 			<?php if($page>1){ ?>
-				<a href="A2B_notification.php?page=<?php echo $page-1; ?>"> &lt; <?php echo gettext("Newer") ?> </a>
+				<a href="A2B_notification__vt.php?page=<?php echo $page-1; ?>"> &lt; <?php echo gettext("Newer") ?> </a>
 			<?php } ?>
 			&nbsp;
 			</td>
 			<td colspan="3" align="right">
 			&nbsp;
                          <?php if($page<$nb_page){ ?>
-			<a href="A2B_notification.php?page=<?php echo $page+1; ?>"><?php echo gettext("Older") ?> &gt;</a>
+			<a href="A2B_notification__vt.php?page=<?php echo $page+1; ?>"><?php echo gettext("Older") ?> &gt;</a>
                         <?php } ?>
 			</td>
 		</tr>
@@ -267,14 +268,14 @@
 $(document).ready(function () {
 
 	$('.newrecord').click(function () {
-		$.get("A2B_notification.php", { id: ""+ this.id, action: "view" },
+		$.get("A2B_notification__vt.php", { id: ""+ this.id, action: "view" },
 			  function(data){
 			    if(data=="true") location.reload(true);
 			  });			
         });
 
 	$('#viewall').click(function () {
-		$.get("A2B_notification.php", { page: ""+ page, action: "viewall", ids : ids },
+		$.get("A2B_notification__vt.php", { page: ""+ page, action: "viewall", ids : ids },
 			  function(data){
 			    if(data=="true") location.reload(true);
 			  });
@@ -282,7 +283,7 @@
 
 	$('.delete').click(function () {
 			if (confirm("<?php echo gettext("Do you want delete this notification ?") ?>")) { 
-				$.get("A2B_notification.php", { id: ""+ this.id, action: "delete" },
+				$.get("A2B_notification__vt.php", { id: ""+ this.id, action: "delete" },
 					  function(data){
 					      location.reload(true);
 					  });	
