--- admin/Public/A2B_entity_password.php	2010-04-07 14:32:20.000000000 +0400
+++ vt_simplified_gui_1.7.0/admin/Public/A2B_entity_password__vt.php	2010-04-08 20:04:29.000000000 +0400
@@ -32,14 +32,14 @@
 **/
 
 
-include ("../lib/admin.defines.php");
-include ("../lib/admin.module.access.php");
-include ("../lib/Form/Class.FormHandler.inc.php");
-include ("../lib/admin.smarty.php");
+include ("../lib/admin.defines__vt.php");
+include ("../lib/admin.module.access__vt.php");
+include ("../lib/Form/Class.FormHandler.inc__vt.php");
+include ("../lib/admin.smarty__vt.php");
 
 if (!$ACXACCESS) {
 	Header("HTTP/1.0 401 Unauthorized");
-	Header("Location: PP_error.php?c=accessdenied");
+	Header("Location: PP_error__vt.php?c=accessdenied");
 	die();
 }
 
@@ -69,7 +69,7 @@
 }
 
 // #### HEADER SECTION
-$smarty->display('main.tpl');
+$smarty->display('main__vt.tpl');
 ?>
 <script language="JavaScript">
 function CheckPassword()
