--- admin/Public/form_data/FG_var_monitor.inc	2010-04-07 14:32:18.000000000 +0400
+++ vt_simplified_gui_1.7.0/admin/Public/form_data/FG_var_monitor__vt.inc	2010-04-08 20:04:29.000000000 +0400
@@ -44,7 +44,7 @@
 
 
 $HD_Form ->FG_LIST_ADDING_BUTTON1 = true;
-$HD_Form ->FG_LIST_ADDING_BUTTON_LINK1 = "A2B_entity_monitor.php?form_action=ask-add&section=".$_SESSION["menu_section"];
+$HD_Form ->FG_LIST_ADDING_BUTTON_LINK1 = "A2B_entity_monitor__vt.php?form_action=ask-add&section=".$_SESSION["menu_section"];
 $HD_Form ->FG_LIST_ADDING_BUTTON_ALT1 = $HD_Form ->FG_LIST_ADDING_BUTTON_MSG1 = gettext("Add Monitoring");
 $HD_Form ->FG_LIST_ADDING_BUTTON_IMG1 = Images_Path ."/map.png" ;
 
