--- admin/Public/form_data/FG_var_def_ratecard.inc	2010-04-07 14:32:18.000000000 +0400
+++ vt_simplified_gui_1.7.0/admin/Public/form_data/FG_var_def_ratecard__vt.inc	2010-04-08 20:04:29.000000000 +0400
@@ -102,7 +102,12 @@
 $HD_Form->FG_FILTER_SEARCH_SESSION_NAME = 'entity_ratecard_selection';
 
 if (!$popup_select) {
-    $HD_Form ->FG_LIST_ADDING_BUTTON1 = true;
+    if (VT_MODE) 
+    { 
+    $HD_Form ->FG_LIST_ADDING_BUTTON1 = false; 
+    } else {
+    $HD_Form ->FG_LIST_ADDING_BUTTON1 = true; 
+    }
     $HD_Form ->FG_LIST_ADDING_BUTTON_LINK1 = "A2B_entity_def_ratecard.php?form_action=ask-add&atmenu=ratecard&section=".$_SESSION["menu_section"];
     $HD_Form ->FG_LIST_ADDING_BUTTON_ALT1 = $HD_Form ->FG_LIST_ADDING_BUTTON_MSG1 = gettext("Add Rate");
     $HD_Form ->FG_LIST_ADDING_BUTTON_IMG1 = Images_Path ."/world_add.png" ;
@@ -153,14 +158,27 @@
 } else {
 	$HD_Form -> FieldExportElement (RATE_EXPORT_FIELD_LIST);
 	$HD_Form -> FG_EXPORT_CSV = true;
+	if (VT_MODE) { $HD_Form -> FG_EXPORT_XML = false;
+	} else {
 	$HD_Form -> FG_EXPORT_XML = true;
+	}
 	$HD_Form -> FG_EXPORT_SESSION_VAR = "pr_export_entity_rates";
+	if (VT_MODE) {	
+	$HD_Form -> FG_EDITION = false;
+	$HD_Form -> FG_DELETION = false;
+	$HD_Form -> FG_ADDITION = false;
+	} else {
 	$HD_Form -> FG_EDITION = true;
 	$HD_Form -> FG_DELETION = true;
 	$HD_Form -> FG_ADDITION = true;
+	}	
 }
 
+if (VT_MODE) {
+$HD_Form -> CV_NO_FIELDS  = '';
+} else { 
 $HD_Form -> CV_NO_FIELDS  = gettext("NO")." ".strtoupper($HD_Form->FG_INSTANCE_NAME)." ".gettext("HAVE BEEN CREATED."); 
+}
 $HD_Form -> CV_DISPLAY_LINE_TITLE_ABOVE_TABLE = false;
 $HD_Form -> CV_TEXT_TITLE_ABOVE_TABLE = '';
 $HD_Form -> CV_DISPLAY_FILTER_ABOVE_TABLE = false;
@@ -172,16 +190,18 @@
 $HD_Form -> FG_FILTER_SEARCH_1_TIME_TEXT = gettext("START DATE");
 $HD_Form -> FG_FILTER_SEARCH_1_TIME_FIELD = 'startdate';
 
-
+if (!VT_MODE) {
 $HD_Form -> AddSearchElement_C1(gettext("TAG"), 'tag','tagtype');
+}
 $HD_Form -> AddSearchElement_C1(gettext("DIALING PREFIX"),'dialprefix','dialprefixtype');
 $HD_Form -> AddSearchElement_C2(gettext("BUYRATE"),'buyrate1','buyrate1type','buyrate2','buyrate2type','buyrate');
 $HD_Form -> AddSearchElement_C2(gettext("RATE INITIAL"),'rateinitial1','rateinitial1type','rateinitial2','rateinitial2type','rateinitial');
 
+if (!VT_MODE) {
 $HD_Form -> FG_FILTER_SEARCH_FORM_SELECT_TEXT = '';
 $HD_Form -> AddSearchElement_Select('SELECT TRUNK', "cc_trunk", "id_trunk, trunkcode, providerip", "", "trunkcode", "ASC", "id_trunk");
 $HD_Form -> AddSearchElement_Select('SELECT RATECARD', "cc_tariffplan", "id, tariffname", "", "tariffname", "ASC", "idtariffplan", 1, null);
-
+}
 
 $HD_Form -> AddEditElement(gettext("RATECARD"),
    "idtariffplan",	
