--- admin/Public/form_data/FG_var_card.inc	2010-04-07 14:32:18.000000000 +0400
+++ vt_simplified_gui_1.7.0/admin/Public/form_data/FG_var_card__vt.inc	2010-04-08 20:04:29.000000000 +0400
@@ -64,13 +64,20 @@
 }
 
 if (!($popup_select>=1)) {
-	$HD_Form ->FG_LIST_ADDING_BUTTON1 = true;
-	$HD_Form ->FG_LIST_ADDING_BUTTON_LINK1 = "A2B_entity_card_multi.php?stitle=Card&section=".$_SESSION["menu_section"];
+	if (VT_MODE) { 
+    $HD_Form ->FG_LIST_ADDING_BUTTON1 = false;
+} else { 
+    $HD_Form ->FG_LIST_ADDING_BUTTON1 = true;
+}
+	$HD_Form ->FG_LIST_ADDING_BUTTON_LINK1 = "A2B_entity_card_multi__vt.php?stitle=Card&section=".$_SESSION["menu_section"];
 	$HD_Form ->FG_LIST_ADDING_BUTTON_ALT1 = $HD_Form ->FG_LIST_ADDING_BUTTON_MSG1 = gettext("Generate Customers");
 	$HD_Form ->FG_LIST_ADDING_BUTTON_IMG1 = Images_Path ."/user_multi_add.png" ;
-	
-	$HD_Form ->FG_LIST_ADDING_BUTTON2 = true;
-	$HD_Form ->FG_LIST_ADDING_BUTTON_LINK2 = "A2B_entity_card.php?form_action=ask-add&atmenu=card&stitle=Card&section=".$_SESSION["menu_section"];
+    if (VT_MODE) {  
+	$HD_Form ->FG_LIST_ADDING_BUTTON2 = false;
+} else {
+    $HD_Form ->FG_LIST_ADDING_BUTTON2 = true;
+}
+	$HD_Form ->FG_LIST_ADDING_BUTTON_LINK2 = "A2B_entity_card__vt.php?form_action=ask-add&atmenu=card&stitle=Card&section=".$_SESSION["menu_section"];
 	$HD_Form ->FG_LIST_ADDING_BUTTON_ALT2 = $HD_Form ->FG_LIST_ADDING_BUTTON_MSG2 = gettext("Add Customer");
 	$HD_Form ->FG_LIST_ADDING_BUTTON_IMG2 = Images_Path ."/user_add.png" ;
 }
@@ -94,9 +101,9 @@
 }
 
 $restriction_list = Constants::getRestrictionList();
-$cardstatus_list = Constants::getCardStatus_List();
-$cardstatus_list_r = Constants::getCardStatus_Revert_List();
-$cardstatus_list_acronym = Constants::getCardStatus_Acronym_List();
+$cardstatus_list = Constants::getCardStatus_VT_List();
+$cardstatus_list_r = Constants::getCardStatus_VT_Revert_List();
+$cardstatus_list_acronym = Constants::getCardStatus_VT_Acronym_List();
 $typepaid_list = Constants::getPaidTypeList();
 $typepaid_list_r = Constants::getPaidTypeList_Revert_List();
 $expire_list = Constants::getCardExpire_List();
@@ -208,7 +215,16 @@
 // Code here for adding the fields in the Export File
 $HD_Form -> FieldExportElement(CARD_EXPORT_FIELD_LIST);
 if (!($popup_select>=1)) $HD_Form -> FG_EXPORT_CSV = true;
-if (!($popup_select>=1)) $HD_Form -> FG_EXPORT_XML = true;
+
+
+if (!($popup_select>=1))  {
+    if (VT_MODE) { 
+    $HD_Form -> FG_EXPORT_XML = false;
+    } else {
+    $HD_Form -> FG_EXPORT_XML = true;
+    }
+}
+
 $HD_Form -> FG_EXPORT_SESSION_VAR = "pr_export_entity_card";
 
 
@@ -251,8 +267,13 @@
 $HD_Form -> AddSearchElement_Select(gettext("SELECT PAYMENT TYPE"), null, null, null, null, null, "typepaid", 0, $typepaid_list_r);
 
 if (($popup_select>=1)) {
-	$HD_Form -> FG_OTHER_BUTTON1 = true;
-	$HD_Form -> FG_OTHER_BUTTON1_ALT = '<font color="red">&lt;select&gt;</font>';
+//This disables the select botton in the CDR
+    if (VT_MODE){
+    $HD_Form -> FG_OTHER_BUTTON1 = true;
+    } else { 
+    $HD_Form -> FG_OTHER_BUTTON1 = true;
+    }
+    $HD_Form -> FG_OTHER_BUTTON1_ALT = '<font color="red">&lt;select&gt;</font>';
 	$HD_Form -> FG_OTHER_BUTTON1_IMG = '';
 
 	if ($popup_select==1) {
@@ -263,11 +284,19 @@
 
 } else {
 	$HD_Form -> FG_INFO = true;
+     if (VT_MODE){    
+	$HD_Form -> FG_OTHER_BUTTON1 = false;
+	$HD_Form -> FG_OTHER_BUTTON2 = false;
+	$HD_Form -> FG_OTHER_BUTTON3 = false;
+	$HD_Form -> FG_OTHER_BUTTON4 = false;
+	$HD_Form -> FG_OTHER_BUTTON5 = false;
+     } else {
 	$HD_Form -> FG_OTHER_BUTTON1 = true;
 	$HD_Form -> FG_OTHER_BUTTON2 = true;
 	$HD_Form -> FG_OTHER_BUTTON3 = true;
 	$HD_Form -> FG_OTHER_BUTTON4 = true;
 	$HD_Form -> FG_OTHER_BUTTON5 = true;
+     }
 	$FN_COLUMN_ID="col".($FG_COL_QUERY_COUNT);//get numbers of rows for needed fields
 	$FN_COLUMN_USERNAME="col".($FG_COL_QUERY_COUNT+1);
 	$FN_COLUMN_ALIAS="col".($FG_COL_QUERY_COUNT+2);
@@ -276,7 +305,7 @@
 	$HD_Form -> FG_OTHER_BUTTON3_LINK="A2B_entity_payment.php?form_action=ask-add&card_id=|$FN_COLUMN_ID|";
 	$HD_Form -> FG_OTHER_BUTTON4_LINK="A2B_entity_subscriber.php?form_action=ask-add&id=|$FN_COLUMN_ID|";
 	$HD_Form -> FG_OTHER_BUTTON5_LINK="A2B_mass_mail.php?form_action=ask-add&id=|$FN_COLUMN_ID|";
-	$HD_Form -> FG_INFO_LINK = "A2B_card_info.php?id=";
+	$HD_Form -> FG_INFO_LINK = "A2B_card_info__vt.php?id=";
 	//FOR INFO , DEL, EDIT  the image is already in Image_Path
 	$HD_Form -> FG_OTHER_BUTTON1_IMG = Images_Path . "/icon-sip.gif";
 	$HD_Form -> FG_OTHER_BUTTON2_IMG = Images_Path . "/icon-iax.gif";
@@ -304,8 +333,20 @@
 $HD_Form -> FG_ADDITION = true;
 
 if (has_rights (ACX_MODIFY_CUSTOMERS) && !($popup_select>=1)) {
-	$HD_Form -> FG_EDITION = true;
-	$HD_Form -> FG_DELETION = true;
+
+    if (VT_MODE) { 
+    $HD_Form -> FG_EDITION = true;
+} else {
+    $HD_Form -> FG_EDITION = false;
+}
+
+    if (VT_MODE) {
+    $HD_Form -> FG_DELETION = false;
+} else {
+    $HD_Form -> FG_DELETION = true;
+}
+
+
 	
 	$maxi	= '';
 	$maxi2	= '';
@@ -331,24 +372,24 @@
 				   "" , "", "",	"", "" , "", "", '', null, null, gettext("Customer Information"));
 				   
 	
-	$HD_Form -> AddEditElement (gettext("WEBUI LOGIN"),
+	$HD_Form -> AddEditElement (gettext("VOIP LOGIN"),
 				   "useralias",
 				   "",
 				   "INPUT",
-				    "size=20 $maxi2 maxlength=40",
+				    "size=20 $maxi2 readonly maxlength=40",
 				   "4",
 				   gettext("Insert the webui login"),
 				   "" , "", "",	"", "" , "", "", "");
 	
 	
-	$HD_Form -> AddEditElement (gettext("WEBUI PASSWORD"),
+	$HD_Form -> AddEditElement (gettext("VOIP PASSWORD"),
 				   "uipass",
 				   '',
 				   "INPUT",
-				   "size=20 $maxi3 maxlength=20",
+				   "size=20 $maxi3 readonly maxlength=20",
 				   "",
 				   gettext("Insert the password"),
-				   "" , "", "",	"", "", "",	"", gettext("Password for customer to access to the web interface and view the balance.") );
+				   "" , "", "",	"", "", "",	"", gettext("Password for customer to register with the VoIP gateway") );
 	
 	if ($form_action =="ask-add" || $form_action=="add") {
 		$balance_access = "value = '0'";
@@ -363,12 +404,12 @@
 				   "credit",
 				   "",
 				   "INPUT",
-				   "size=30 maxlength=30 ".$balance_access,
+				   "size=30 readonly maxlength=30 ".$balance_access,
 				   "12",
 				   gettext("Insert the number of credit"),
 				   "" , "", "", "", "", "", "", gettext("currency : ").strtoupper(BASE_CURRENCY) );
-	
 				   
+if (!VT_MODE) {
 	$HD_Form -> AddEditElement (gettext("CUSTOMER GROUP"),
 				   "id_group",
 				   "",
@@ -378,19 +419,18 @@
 				   "cc_card_group",
 				   "name, id",
 				   "", "", "%1", "", "", "", "", "");
-			   
+
 	$HD_Form -> AddEditElement (gettext("SERIAL"),
-	               "id_seria",
-	               "",
-	               "SELECT",
-	               "", "", "",
-	               "sql",
-	               "cc_card_seria",
-	               "name, id",
-	               "", "", "%1", "", "", "", '<OPTION  value="-1" selected>'.gettext("NOT DEFINED").'</OPTION>' );
-	
-	
-	$HD_Form -> AddEditElement (gettext("LAST NAME"),
+		                   "id_seria",
+	        	           "",
+	               		   "SELECT",
+	               		   "", "", "",
+	               		   "sql",
+	               		   "cc_card_seria",
+	               		   "name, id",
+	               		   "", "", "%1", "", "", "", '<OPTION  value="-1" selected>'.gettext("NOT DEFINED").'</OPTION>' );
+}	
+        $HD_Form -> AddEditElement (gettext("LAST NAME"),
 				   "lastname",
 				   '$value',
 				   "INPUT",
@@ -523,7 +563,7 @@
 					   "",
 					   "list" , "", "",	"", $typepaid_list, "%1", "", '', null, null, gettext("Customer Status") );
 	}
-	
+if (!VT_MODE) {	
 	$HD_Form -> AddEditElement (gettext("CALL PLAN"),
 				   "tariff",
 				   "",
@@ -534,7 +574,6 @@
 				   "tariffgroupname, id",
 				   "", "", "%1", "", gettext("Changing the call plan will result in the free minutes or free calls package being reset.") );
 	
-	
 	$HD_Form -> AddEditElement (gettext("DIDGROUP"),
 				   "id_didgroup",
 				   "",
@@ -545,7 +584,7 @@
 				   "didgroupname, id",
 				   "", "", "%1", "", "", "", '<OPTION  value="-1" selected>'.gettext("NOT DEFINED").'</OPTION>' );
 	
-	$timezone_list = get_timezones();
+    $timezone_list = get_timezones();
 	$one_select = false;
 	foreach ($timezone_list as $key => $cur_value) {
 		$timezone_list[$key] = array($cur_value[2], $key);
@@ -564,6 +603,7 @@
 				   "","","",
 				   "list" , "", "",	"", $timezone_list, "%1", "", "" );
 	
+}
 	
 	$HD_Form -> AddEditElement (gettext("LANGUAGE"),
 				   "language",
@@ -574,7 +614,7 @@
 				   "",
 				   "list" , "", "",	"", $language_list, "%1", "", "" );
 	
-	
+
 	$HD_Form -> AddEditElement (gettext("CURRENCY"),
 				   "currency",
 				   "",
@@ -625,7 +665,7 @@
 				   "",
 				   "list" , "", "",	"", $simultaccess_list, "%1", "", "" );
 	
-	
+if (!VT_MODE) {	
 	$HD_Form -> AddEditElement (gettext("RUN SERVICE"),
 				   "runservice",
 				   '0',
@@ -635,7 +675,6 @@
 				   gettext("Choose if you want to enable the service recurring on this account"),
 				   "" , "", "", "Yes :1, - No:0", "", "" , "", gettext("Apply recurring service to this account."));
 	
-	
 	$HD_Form -> AddEditElement (gettext("CREDIT LIMIT"),
 				   "creditlimit",
 				   "",
@@ -707,7 +746,8 @@
 				   "cc_campaign",
 				   "name, id",
 				   "", "", "%1", "", "", "", '<OPTION  value="-1" selected>'.gettext("NOT DEFINED").'</OPTION>' );
-	
+//END OF VT_MODE IF CLAUSE	
+}	
 	
 	$HD_Form -> AddEditElement (gettext("FIRST USE DATE"),
 				   "firstusedate",
@@ -758,7 +798,7 @@
 				   "4",
 				   gettext("Insert the number of days after which the account will expire"),
 				   "" , "", "",	"", "", "", "", gettext("The number of days after which the account will expire."));
-	
+if (!VT_MODE) {	
 	
 	$HD_Form -> AddEditElement (gettext("CREATE SIP CONFIG"),
 				   "sip_buddy",
@@ -778,7 +818,8 @@
 				   "",
 				   gettext("Choose if you want to enable the iax account"),
 				   "" , "", "" , "Yes :1, - No:0",  "", "" , "", gettext("Create the IAX config automatically"));
-	
+}
+
 	$HD_Form -> AddEditElement (gettext("MAC ADDRESS"),
 				   "mac_addr",
 				   '$value',
@@ -797,7 +838,7 @@
 				   "",
 				   "",
 				   "" , "", "",	"", "", "", "NO", gettext("Updated to show the number of concurrent calls in use by this customer. If there are no currently no calls, and the system shows that there are, manually reset this field back to zero.") );
-	
+if (!VT_DEMO) {	
 	$HD_Form -> AddEditElement (gettext("AUTOREFILL"),
 				   "autorefill",
 				   '0',
@@ -828,13 +869,13 @@
 				   "",
 				   "list" , "", "",	"", $invoiceday_list, "%1", "", gettext("Define the day of the month when the system will generate the customer invoice."), 
 				   '', '', gettext("Invoice Status"));
-	
+}	
 	
 	if ($form_action =="ask-add" || $form_action=="add") {
 		$vat_init = "value = '0'";
 		
 	}
-	
+if (!VT_DEMO) {	
 	$HD_Form -> AddEditElement (gettext("VAT"),
 				   "vat",
 				   "",
@@ -907,7 +948,6 @@
 				   "",
 				   "", "", "", "", "", "cc_restricted_phonenumber:number:number, id_card:id_card = %id::id_card:::::::0");
 	
-	
 				   
 	$HD_Form -> AddEditElement (gettext("CALLERID"),
 				   "callerid",
@@ -925,6 +965,7 @@
 				   
 	
 	
+}	
 	
 	$HD_Form -> AddEditElement (gettext("CUSTOMER HISTORY"),
 				   "description",
@@ -952,6 +993,9 @@
 	if (ACT_VOICEMAIL)
 		$add_fields_voicemail = 'voicemail_permitted, voicemail_activated, ';
 	
+/* THIS FORM IS EVIL! 	
+
+
 	$HD_Form -> FieldEditElement ('username, useralias, uipass, credit, id_group, id_seria, lastname, firstname, email, '.
 			' address, city, state, country, zipcode, phone, fax,company_name, company_website, '.
 			$update_field_typepaid.' tariff, id_didgroup, id_timezone, language, currency, status, block, lock_pin, simultaccess,  runservice, creditlimit, ' .
@@ -959,6 +1003,11 @@
 			' id_campaign, ' .
 			' firstusedate, enableexpire, expirationdate, expiredays, sip_buddy, iax_buddy, mac_addr, inuse, ' .
 			' autorefill, initialbalance, invoiceday, vat, vat_rn, discount, traffic, traffic_target, restriction' );
+*/
+        $HD_Form -> FieldEditElement ('username, useralias, uipass, credit, lastname, firstname, email, '.
+             ' address, city, state, country, zipcode, phone, fax, company_name, company_website, '.
+                    $update_field_typepaid.' language, currency, status, block, lock_pin, simultaccess, ' .
+             ' firstusedate, enableexpire, expirationdate, expiredays, mac_addr, inuse' );
 	
 	
 	if(!isset($myusername)) 
@@ -982,7 +1031,7 @@
 	
 	
 	$HD_Form -> FG_BUTTON_EDITION_SRC = $HD_Form -> FG_BUTTON_ADITION_SRC  = Images_Path . "/cormfirmboton.gif";
-	$HD_Form -> FG_BUTTON_EDITION_BOTTOM_TEXT = $HD_Form -> FG_BUTTON_ADITION_BOTTOM_TEXT = gettext("Once you have completed the form above, click on Confirm Data");
+		$HD_Form -> FG_BUTTON_EDITION_BOTTOM_TEXT = $HD_Form -> FG_BUTTON_ADITION_BOTTOM_TEXT = gettext("Once you have completed the form above, click on Confirm Data");
 	
 	$HD_Form -> FG_ADDITIONAL_FUNCTION_BEFORE_EDITION = 'change_card_lock';
 	
