--- admin/Public/form_data/FG_var_voucher.inc	2010-04-07 14:32:18.000000000 +0400
+++ vt_simplified_gui_1.7.0/admin/Public/form_data/FG_var_voucher__vt.inc	2010-04-08 20:04:29.000000000 +0400
@@ -63,13 +63,20 @@
 }
 
 
-
+if (VT_MODE) {
+$HD_Form ->FG_LIST_ADDING_BUTTON1 = false;
+} else { 
 $HD_Form ->FG_LIST_ADDING_BUTTON1 = true;
+}
 $HD_Form ->FG_LIST_ADDING_BUTTON_LINK1 = "A2B_entity_voucher_multi.php?section=".$_SESSION["menu_section"];
 $HD_Form ->FG_LIST_ADDING_BUTTON_ALT1 = $HD_Form ->FG_LIST_ADDING_BUTTON_MSG1 = gettext("Generate Voucher");
 $HD_Form ->FG_LIST_ADDING_BUTTON_IMG1 = Images_Path ."/cog_add.png" ;
 
+if (VT_MODE) {
+$HD_Form ->FG_LIST_ADDING_BUTTON2 = false;
+} else {
 $HD_Form ->FG_LIST_ADDING_BUTTON2 = true;
+}
 $HD_Form ->FG_LIST_ADDING_BUTTON_LINK2 = "A2B_entity_voucher.php?form_action=ask-add&section=".$_SESSION["menu_section"];
 $HD_Form ->FG_LIST_ADDING_BUTTON_ALT2 = $HD_Form ->FG_LIST_ADDING_BUTTON_MSG2 = gettext("Add Voucher");
 $HD_Form ->FG_LIST_ADDING_BUTTON_IMG2 = Images_Path ."/add.png" ;
@@ -106,19 +113,32 @@
 $HD_Form -> AddSearchElement_Select (gettext("SELECT CURRENCY"), null, null, null, null, null, "currency", 0, $currency_list_r);
 
 
-
+if (VT_MODE) {
+$HD_Form -> CV_NO_FIELDS  = '';
+} else {
 $HD_Form -> CV_NO_FIELDS  = gettext("NO")." ".strtoupper($HD_Form->FG_INSTANCE_NAME)." ".gettext("HAVE BEEN CREATED.");
+}
 $HD_Form -> CV_DISPLAY_LINE_TITLE_ABOVE_TABLE = false;
 $HD_Form -> CV_TEXT_TITLE_ABOVE_TABLE = '';
 $HD_Form -> CV_DISPLAY_FILTER_ABOVE_TABLE = false;
 $HD_Form -> FG_EXPORT_SESSION_VAR = "pr_export_entity_voucher";
 $HD_Form -> FG_EXPORT_FIELD_LIST = VOUCHER_EXPORT_FIELD_LIST;
 $HD_Form -> FG_EXPORT_CSV = true;
+if (VT_MODE) {
+$HD_Form -> FG_EXPORT_XML = false;
+} else {
 $HD_Form -> FG_EXPORT_XML = true;
+}
 
+if (VT_MODE) {
+$HD_Form -> FG_EDITION = false;
+$HD_Form -> FG_DELETION = false;
+$HD_Form -> FG_ADDITION = false;
+} else {
 $HD_Form -> FG_EDITION = true;
 $HD_Form -> FG_DELETION = true;
 $HD_Form -> FG_ADDITION = true;
+}
 
 if ($form_action=="ask-add") {
 	$card_gen = gen_card($HD_Form->FG_TABLE_NAME, LEN_VOUCHER, "voucher");
