--- admin/Public/templates/default/index.tpl	2010-04-07 14:32:20.000000000 +0400
+++ vt_simplified_gui_1.7.0/admin/Public/templates/default/index__vt.tpl	2010-04-08 20:04:29.000000000 +0400
@@ -36,7 +36,7 @@
 
 {/literal}
 
-	<form name="form" method="POST" action="PP_intro.php" onsubmit="return test()">
+	<form name="form" method="POST" action="PP_intro__vt.php" onsubmit="return test()">
 	<input type="hidden" name="done" value="submit_log">
 
 	
@@ -104,7 +104,7 @@
 <script LANGUAGE="JavaScript">
 	document.form.pr_login.focus();
         $("#ui_language").change(function () {
-          self.location.href= "index.php?ui_language="+$("#ui_language option:selected").val();
+          self.location.href= "index__vt.php?ui_language="+$("#ui_language option:selected").val();
         });
 </script>
 {/literal}
