--- Public/index.php	2009-11-02 09:23:15.000000000 +0100
+++ Public/index__vt.php	2009-11-02 09:34:52.000000000 +0100
@@ -32,13 +32,13 @@
 **/
 
 
-include ("../lib/admin.defines.php");
-include ("../lib/admin.smarty.php");
+include ("../lib/admin.defines__vt.php");
+include ("../lib/admin.smarty__vt.php");
 
 getpost_ifset(array (
 	'error'
 ));
 
 $smarty -> assign("error", $error);
-$smarty -> display('index.tpl');
+$smarty -> display('index__vt.tpl');
 
