--- Public/PP_intro.php	2009-11-02 11:22:52.000000000 +0100
+++ Public/PP_intro__vt.php	2009-11-02 15:20:05.000000000 +0100
@@ -32,18 +32,18 @@
 **/
 
 
-include_once ("../lib/admin.defines.php");
-include_once ("../lib/admin.module.access.php");
-include_once ("../lib/admin.smarty.php");
+include_once ("../lib/admin.defines__vt.php");
+include_once ("../lib/admin.module.access__vt.php");
+include_once ("../lib/admin.smarty__vt.php");
 
 
 if (!$ACXACCESS) {
 	Header ("HTTP/1.0 401 Unauthorized");
-	Header ("Location: PP_error.php?c=accessdenied");	   
+	Header ("Location: PP_error__vt.php?c=accessdenied");	   
 	die();	   
 }
 
-$smarty->display('main.tpl');
+$smarty->display('main__vt.tpl');
 
 
 ?>
