--- common/lib/admin.defines.php	2010-04-08 19:37:54.000000000 +0400
+++ vt_simplified_gui_1.7.0/common/lib/admin.defines__vt.php	2010-04-08 20:05:15.000000000 +0400
@@ -53,7 +53,7 @@
 include_once (FSROOT."lib/mail/class.phpmailer.php");
 
 // INCLUDE MISC
-include (FSROOT."lib/Misc.php");
+include (FSROOT."lib/Misc__vt.php");
 include (dirname(__FILE__)."/Class.NotificationsDAO.php");
 include (dirname(__FILE__)."/Class.Notification.php");
 include (dirname(__FILE__)."/Class.Mail.php");
@@ -105,8 +105,11 @@
 // Define a demo mode
 define("DEMO_MODE", false);
 
+// Define VT mode
+define("VT_MODE", true);
+
 // Parameter to show link to Asterisk GUI
-define("ASTERISK_GUI_LINK", false);
+define("ASTERISK_GUI_LINK", true);
 
 
 define ("LEN_ALIASNUMBER", isset($A2B->config['global']['len_aliasnumber'])?$A2B->config['global']['len_aliasnumber']:null);
@@ -219,7 +222,7 @@
 // INCLUDE HELP
 include (LIBDIR."admin.help.php");
 
-include (LIBDIR."common.defines.php");
+include (LIBDIR."common.defines__vt.php");
 
 
 // COPYRIGHT
