--- common/lib/admin.smarty.php	2010-04-01 11:12:39.000000000 +0400
+++ vt_simplified_gui_1.7.0/common/lib/admin.smarty__vt.php	2010-04-08 20:05:15.000000000 +0400
@@ -62,7 +62,6 @@
 }
 $smarty->assign("popupwindow", $popup_select);
 
-
 if (!empty($msg)) {
 	switch($msg) {
 		case "nodemo": 	$smarty->assign("MAIN_MSG", '<center><b><font color="red">'.gettext("This option is not available on the Demo!").'</font></b></center><br>');
@@ -93,6 +92,7 @@
 
 $smarty->assign("HTTP_HOST", $_SERVER['HTTP_HOST']);
 $smarty->assign("ASTERISK_GUI_LINK", ASTERISK_GUI_LINK);
+$smarty->assign("VT_MODE", VT_MODE);
 
 $smarty->assign("LCMODAL", LCMODAL);
 
