--- common/lib/admin.module.access.php	2010-04-01 11:12:39.000000000 +0400
+++ vt_simplified_gui_1.7.0/common/lib/admin.module.access__vt.php	2010-04-08 20:05:15.000000000 +0400
@@ -76,7 +76,7 @@
 	session_destroy();
 	$rights=0;
 	Header ("HTTP/1.0 401 Unauthorized");
-	Header ("Location: index.php");
+	Header ("Location: index__vt.php");
 	die();
 }
 
@@ -116,7 +116,7 @@
 
 		if (!is_array($return) || $return[1]==0 ) {
 			header ("HTTP/1.0 401 Unauthorized");
-			Header ("Location: index.php?error=1");
+			Header ("Location: index__vt.php?error=1");
 			die();
 		}
 		// if groupID egal 1, this user is a root
