--- common/lib/common.defines.php	2010-04-08 20:34:08.000000000 +0400
+++ vt_simplified_gui_1.7.0/common/lib/common.defines__vt.php	2010-04-08 20:35:45.000000000 +0400
@@ -127,7 +127,7 @@
 define ("EMAILCONTACT", "areski@gmail.com");
 
 // A2BILLING INFO  
-define ("COPYRIGHT", "A2Billing 1.7.0 (Larch), A2Billing is software licensed under the ".'<a href="http://www.fsf.org/licensing/licenses/agpl-3.0.html" target="_blank">AGPL 3</a>' . ". <br/>" . "Copyright (C) 2004-2010 - Star2billing S.L. <a href=\"http://www.star2billing.com\" target=\"_blank\">http://www.star2billing.com/</a>");
+define ("COPYRIGHT", "A2Billing 1.7.0 (Larch VT), A2Billing is software licensed under the ".'<a href="http://www.fsf.org/licensing/licenses/agpl-3.0.html" target="_blank">AGPL 3</a>' . ". <br/>" . "Copyright (C) 2004-2010 - Star2billing S.L. <a href=\"http://www.star2billing.com\" target=\"_blank\">http://www.star2billing.com/</a>");
 
 define ("CCMAINTITLE", gettext("A2Billing Portal"));
 
