--- common/lib/Misc.php	2010-04-01 11:12:39.000000000 +0400
+++ vt_simplified_gui_1.7.0/common/lib/Misc__vt.php	2010-04-08 20:05:15.000000000 +0400
@@ -488,7 +488,7 @@
 	$list_customer = $inst_table->Get_list($handle, $FG_TABLE_CLAUSE, "", "", "", "", "", "", "", 10);
 	$id = $list_customer[0][0];
 	if ($id > 0) {
-		echo "<a href=\"A2B_entity_card.php?form_action=ask-edit&id=$id\">$value</a>";
+		echo "<a href=\"A2B_entity_card__vt.php?form_action=ask-edit&id=$id\">$value</a>";
 	} else {
 		echo $value;
 	}
